src/components/lsk/lsk.pug

```diff
@@ -1,2 +1,2 @@
-span(ng-bind='$ctrl.amount | lsk')
+span(ng-bind='$ctrl.amount | lsk | number:8')
```

Comment:
`number:8` is also not the answer as we don't want to show the trailing zeros. 
`100.00000000` should be `100`
`100.12300000` should be `100.123`